### PR TITLE
EPMEDU-754: [Android] Add Firebase Crashlytics to project

### PR DESCRIPTION
**Pull request by FlinForm**
image

**Note:** _google-services.json_ can be found on KB page: https://kb.epam.com/display/EPMEDU/%5BEPM-ANML%5D+%5BAndroid%5D+Development

### CODE REVIEW

**Review by Fileek — COMMENTED** [review]
Please attach google-services.json or link to download it

**Review by Fileek — APPROVED** [review]

File: app/build.gradle.kts [hunk 1]
```diff
@@ -20,6 +22,9 @@ dependencies {
     implementation(projects.base)
     implementation(projects.feature.more)
 
+    implementation(platform(libs.firebase.bom))
+    implementation("com.google.firebase:firebase-crashlytics-ktx")
```
Comment (egorikftp) [inline app/build.gradle.kts]: please make in libs.toml

`firebase-crashlytics = { module = "com.google.firebase:firebase-crashlytics" }`

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -19,12 +19,19 @@ kotlin = "1.6.10"
 lifecycle = "2.4.1"
 navigation = "2.4.2"
 accompanist = "0.23.1"
+googleServices = "4.3.10"
```
Comment (egorikftp) [inline gradle/libs.versions.toml]: it will be better to leave sorted as previously)

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -19,12 +19,19 @@ kotlin = "1.6.10"
...
 
 [libraries]
 gradle-plugin-buildtools = { module = "com.android.tools.build:gradle", version.ref = "buildTools" }
 gradle-plugin-detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
 gradle-plugin-hilt = { module = "com.google.dagger:hilt-android-gradle-plugin", version.ref = "hilt" }
 gradle-plugin-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+gradle-plugin-googleservices = { module = "com.google.gms:google-services", version.ref = "googleServices" }
+gradle-plugin-crashlytics = { module = "com.google.firebase:firebase-crashlytics-gradle", version.ref = "crashlytics" }
+
+firebase-bom = { module = "com.google.firebase:firebase-bom", version.ref = "firebaseBom" }
```
Comment (egorikftp) [inline gradle/libs.versions.toml]: the same, please)

File: app/build.gradle.kts (hunk 1 above)
Comment (FlinForm) [reply app/build.gradle.kts]: done

File: gradle/libs.versions.toml (hunk 3 above)
Comment (FlinForm) [reply gradle/libs.versions.toml]: done

File: gradle/libs.versions.toml (hunk 2 above)
Comment (FlinForm) [reply gradle/libs.versions.toml]: done

**Review by egorikftp — APPROVED** [review]

**Review by DoTheMonkeyBusiness — APPROVED** [review]

**Pull request merged**